cookbook/specs/spec_add_blast.json

```diff
@@ -0,0 +1,72 @@
+{
+  "proposal": {
+    "title": "Add Specs: Blast Sepolia",
+    "description": "Adding new specification support for relaying data on Blast Sepolia Testnet",
+    "specs": [
+      {
+        "index": "BLSTSP",
+        "name": "Blast Sepolia testnet",
+        "enabled": true,
+        "imports": ["ETH1"],
+        "reliability_threshold": 268435455,
+        "data_reliability_enabled": true,
+        "block_distance_for_finalized_data": 6,
+        "blocks_in_finalization_proof": 2,
+        "average_block_time": 15000,
+        "allowed_block_lag_for_qos_sync": 10,
+        "shares": 1,
+        "min_stake_provider": {
+          "denom": "ulava",
+          "amount": "50000000000"
+        },
+        "api_collections": [
+          {
+            "enabled": true,
+            "collection_data": {
+              "api_interface": "jsonrpc",
+              "internal_path": "",
+              "type": "POST",
+              "add_on": ""
+            },
+            "apis": [],
+            "headers": [],
+            "inheritance_apis": [],
+            "parse_directives": [],
+            "verifications": [
+              {
+                "name": "chain-id",
+                "values": [
+                  {
+                    "expected_value": "0xA30769"
+                  }
+                ]
+              },
+              {
+                "name": "pruning",
+                "values": [
+                  {
+                    "latest_distance": 1024
```

Comment:
this means any query 1025+ blocks in the past will either need to go to archive or fail, our usual rule of thumb is 24h of blocks so 24*3600/averageBlockTime(s)